### DISCUSSION TITLE
What is your experience with the Message Processing Framework?

### DISCUSSION THREAD

**Discussion by eddiemcs3** [aws/aws-dotnet-messaging#137] [General]
We're looking for feedback on your experience with the Message Processing Framework.  Please share with us and what you would like to see or see more of.

**Comment by brendonparker** [comment]
I plan to play with this very soon and will report back…

I’ve spent most of last week trying to get MassTransit to fit my needs. Largely with using a lambda to process the queue (this proved very difficult when I wanted to handle multiple types from the same queue). Documentation was also very unclear in places. It communicated concepts well, but didn’t “show me the code”. 

Then I started looking for alternatives and found this. Which looks way simpler and in line with what I want.

**Reply by eddiemcs3** [reply]
Thanks, Brendon.  We look forward to your feedback.

**Reply by brendonparker** [reply]
I've started some new discussions (idea) threads. For some of the things I've run into.

- https://github.com/awslabs/aws-dotnet-messaging/discussions/148
- https://github.com/awslabs/aws-dotnet-messaging/discussions/149

**Reply by brendonparker** [reply]
@eddiemcs3 had a few minutes to review the above?

**Reply by eddiemcs3** [reply]
Yes, would like to hear your feedback.
